README.md

```diff
@@ -1 +1,50 @@
-# UnitA Project
+# Unit A landing page
+
+_This project is a landing page for a branch of __APE UNIT__ company which is an international tech company based in Germany_
+
+## Live version
+
+[unit-a.netlify.app](https://unit-a.netlify.app/)
+
+
+## Contents
+
+In this landing page project we have the following sections done:
+
+- [x] Header
+- [x] About
+- [x] Projects
+- [x] Services
+- [x] Available Jobs
+- [x] Call To Action
+- [x] Footer
+
+## Technologies Used
+
+This project is mainly based on [Next.js](https://nextjs.org/) plus others few libraries listed below:
+
+- [Tailwind css](https://tailwindcss.com/)
+- [Classnames](https://www.npmjs.com/package/classnames)
+- [Aos](https://michalsnik.github.io/aos/)
+
+
+## How To Run The Project
+
+First, install the project's dependencies:
+
+```
+npm install
+```
+
+Then, run the development server:
+
+```
+npm run dev
+```
+
+There you go 😊, You can preview the project on [http://localhost:3000](http://localhost:3000)
```

Comment:
Great 👍,
Maybe here you can add more info because the port is dynamic sometimes it is 3001, it's not always 3000